SpatialGDK/Source/SpatialGDK/Private/EngineClasses/AbstractLBStrategy.cpp

```diff
@@ -0,0 +1,22 @@
+// Fill out your copyright notice in the Description page of Project Settings.
```

Comment:
Copyright needs to be set (should also be set in the project settings so you don't need to deal with this).

Reply:
Done